backend/app/api/v1/commons/constants.py

```diff
@@ -0,0 +1,23 @@
+# Define the keywords for sorting.
```

Comment:
Is there a reason this is hard-coded and not retrieved from an API?
What does this apply to?

I think it would be beneficial to mention some of this in the comment.

Reply:
Sure!